api/tests/test_lock.py

```diff
@@ -0,0 +1,97 @@
+import asyncio
```

Comment:
file should be test_lock_manager and follow the same class structure as other test files